.github/ISSUE_TEMPLATE/release.md

```diff
@@ -37,9 +31,13 @@ title: 'vX.Y.Z release'
       go mod tidy && go mod vendor && go mod verify
       git add go.mod go.sum vendor
 
-- [ ] Prepare release notes:
+- [ ] Prepare release notes. You need to generate release notes from both
+      cilium/cilium and cilium/hubble repositories and manually combine them.
```

Comment:
Thankfully, there aren't too many changes on average for a new release. Do we basically rely on the `hubble-cli` label? Is it set automatically for any code that touches the `hubble` directory in the cilium repository?